mmv1/templates/terraform/examples/uptime_check_config_synthetic_monitor.tf.erb

```diff
@@ -0,0 +1,14 @@
+resource "google_monitoring_uptime_check_config" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
This file needs to be added to `examples` of the file [UptimeCheckConfig.yaml](https://github.com/GoogleCloudPlatform/magic-modules/blob/203573897b787f85a6779b8db1a7b050f7e2a39b/mmv1/products/monitoring/UptimeCheckConfig.yaml#L31)

Reply:
done.